templates/csharp/api-plugin-from-scratch-bearer/appPackage/ai-plugin.json.tpl

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://aka.ms/json-schemas/copilot-extensions/v2.1/plugin.schema.json",
+  "schema_version": "v2.1",
+  "name_for_human": "{{appName}}${{APP_NAME_SUFFIX}}",
+  "namespace": "repairs",
+  "description_for_human": "Track your repair records",
+  "description_for_model": "Plugin for searching a repair list, you can search by who's assigned to the repair.",
+  "functions": [
+    {
+      "name": "listRepairs",
+      "description": "Returns a list of repairs with their details and images",
+      "capabilities": {
+        "response_semantics": {
+          "data_path": "$",
```

Comment:
Should be `"data_path: "$.results"`, because the output format of the repair API is something like this:

```json
{
    "results": [
        {
             "id": "..."
        }
    ]
}
```

Reply:
The no-auth template has the same issue, let's fix it later